doc/content/getting-started/aws/ecs/troubleshooting/_index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Troubleshooting AWS ECS Deployment"
+description: ""
+weight: 7
+---
+
+<!--
+TODO: https://github.com/TheThingsNetwork/lorawan-stack/issues/2714
+Move to generic getting started guide once ready.
+-->
+
+This section contains information to troubleshoot {{% tts %}} ECS deployment.
+
+<!--more-->
+
+### TTS no longer accepting connections from UDP gateways after upgrading to latest version
+
+Cross check the below operations
+
+  - Before the version update, complete the database migration.
+  
+  - If `default` is set to the `DefaultTenantId` in the configuration file, unset the `DefaultTenantId` parameter
+  
+  ```
+  tenancy:
+        default-id: 'default'
+  ```
+  Unset the `DefaultTenantId` as below
+  
+  ```
+  tenancy:
+        default-id: ''
+  ```
+
+Apply the configurations and restart the `gs` task.
+
+## Professional Support
+
+Additional paid support for this deployment is offered by The Things Industries. You can contact us either by mailing us at `support@thethingsindustries.com` or by visiting [our support page](https://www.thethingsindustries.com/support/).
```

Comment:
Please only use the support page. The email should be reserved for customers who have already purchased a support plan.